build-tools-internal/performance/elasticsearch-build-logic-change.scenarios

```diff
@@ -0,0 +1,31 @@
+# Can specify scenarios to use when none are specified on the command line
+default-scenarios = ["buildConfigurationNew", "buildConfigurationOld"]
+
+buildConfigurationNew {
+    title = "new"
+    tasks = ["help"]
+    # cleanup-tasks = ["clean"]
+    versions = ["@gradleVersion@"]
+    gradle-args = ["--no-scan", "--no-build-cache"]
+    run-using = cli // value can be "cli" or "tooling-api"
+    daemon = warm // value can be "warm", "cold", or "none"
+    warm-ups = 5
+    iterations = 10
+    git-checkout = {
+        build = "@testGitCommit@"
+    }
+}
+
+buildConfigurationOld {
+    title = "base"
+    tasks = ["help"]
+    versions = ["@gradleVersion@"]
+    gradle-args = ["--no-scan", "--no-build-cache"]
+    run-using = cli // value can be "cli" or "tooling-api"
+    daemon = warm // value can be "warm", "cold", or "none"
+    warm-ups = 5
+    iterations = 10
+    git-checkout = {
+        build = "master"
```

Comment:
I wonder if we should use the PR target branch here?

Reply:
Don't bother with reviewing this yet. Still experimenting and playing around. Pushed to test ci job

Reply:
I changed the scenarios to run 1) against the current commit and 2) against master for comparison reason